config/initializers/0_clowder-config.rb

```diff
@@ -9,7 +9,7 @@
 #  remediations_url = "#{remediations_hostname}:#{remediations_port}"
 
   # compliance-ssg
-  compliance_ssg_config = config.private_dependency_endpoints.dig('compliance-ssg', 'service')
+  compliance_ssg_config = config.dependency_endpoints.dig('compliance-ssg', 'service')
```

Comment:
Can you please clarify this change? The dependent service compliance-ssg uses private web port.

Reply:
Closing ! this was after all not needed, I was using an old version of the compliance-ssg image, sorry :)